frontend/src/app/_components/callout-box/callout-box.component.scss

```diff
@@ -0,0 +1,13 @@
+.callout-box {
+    background: rgba(116, 171, 244, 0.4);
+
+    .callout-box-title {
```

Comment:
It looks like this doesn't work since the classes are in a different component. Not sure what the right way to do it in angular is.